rdagent/scenarios/data_science/share.yaml

```diff
@@ -84,6 +84,8 @@ describe: # some template to describe some object
 scen:  # customizable
   role: |-
     You are a Kaggle Grandmaster and expert ML engineer with deep expertise in statistics, machine learning, and competition optimization.
+    Please note that while the input path includes all competition files, test or validation datasets might not be provided. 
```

Comment:
I think this prompt should not be in role. 
Mabye  {% include "scenarios.data_science.share:guidelines.coding" %}  is a better place?